### PR TITLE
Issue 1 search name - Nimrod Castano

### PR DESCRIPTION
**Pull request by nimcastano**
ajax issue 1

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,16 +3,85 @@
...
   <link rel="stylesheet" href="css/layout.css">
   <link rel="stylesheet" href="css/styles.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Teko:wght@400&display=swap" rel="stylesheet">
 </head>
 <body>
   <header>
+    <div class="header-background">
+      <a class="return"><h1 class="header-name">BEER RUN</h1></a>
```
Comment (shawnkost) [inline index.html]: `a` elements need an `href` attribute. If you don't want it to direct you anywhere, then set the value to a `#`

File: index.html [hunk 2]
```diff
@@ -3,16 +3,85 @@
...
+        <div class="col-full">
+          <img src="./images/beerRunGlasses.png" class="home-photo" alt="glasses of beer on a counter">
+        </div>
+      </div>
+      <div class="row">
+        <div class="col-full">
+          <form class="form-city">
+            <h2 class="city-intro">Find a brewery in your city, or any city.</h2>
+            <label for="city-name">City Name</label>
+            <input type="text" name="city-name" class="textbox">
```
Comment (shawnkost) [inline index.html]: The `input` needs an `id` that matches the `for` of your label. The `for` attribute is linked by `id`, not `name`

File: js/main.js [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+    });
+    xhr.send();
+
+    $homePage.classList.add('hidden');
+    $namePage.classList.remove('hidden');
+
+    if (xhr.response < 10) {
+      document.querySelector('.page-turner').className = 'page-turner hidden';
+    }
+  }
```
Comment (shawnkost) [inline js/main.js]: This should go inside your `load` event

**Review by shawnkost — CHANGES_REQUESTED** [review]
Nicely done Nimrod! Just a few changes requested here

**Comment by nimcastano** [comment]
ajax issue 1

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**